virttest/virsh.py

```diff
@@ -4236,6 +4246,12 @@ def blkdeviotune(name, device=None, options=None,
     Set or get a domain's blkio parameters
```

Comment:
BTW, I think the description of blkdeviotune() is inappropriate, would you please update it? Thanks!

Reply:
the whole blkdeviotune() is not containing enough available virsh api params, will be updated in another pr.

Reply:
It makes sense. LGTM.